package.json

```diff
@@ -25,15 +26,19 @@
     "prettier": "prettier \"**/*.{md,css,html,json,yml}\" --write",
     "test:other": "yarn prettier --list-different",
     "test:js": "eslint --ignore-path .gitignore --ignore-path .prettierignore \"**/*.js\"",
-    "test": "yarn test:other && yarn test:js"
+    "test": "yarn test:other && yarn test:js",
+    "build": "webpack"
```

Comment:
Could it make sense to remove setting mode to `development` in the webpack config and instead add a `dev` script that runs `webpack --mode=development` (leaving the `build` script as is since I believe it runs in `production` mode by default).

Reply:
This would work, but I'd prefer to keep all the configuration in the webpack config unless we need to have multiple build scripts for different modes (currently `production` doesn't work anyway and the only advantage of `none` is speed).